jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunConfigurationTest.kt

```diff
@@ -135,6 +137,29 @@ class SamRunConfigurationTest {
         }
     }
 
+    @Test
+    fun readExternalDoesNotThrowException() {
+        val element = SAXBuilder().build(StringReader("""
```

Comment:
There are extension methods for this in DefaultProjectAccountSettingsManagerTest, if you want to move those into a test util XmlUtils.kt and make them public